src/plugins/oSnap/components/TransactionBuilder/Transaction.vue

```diff
@@ -61,13 +61,20 @@ function setTransactionAsInvalid() {
 </script>
 
 <template>
-  <div class="mt-4 border-b pb-4 first:mt-0">
-    <div class="flex items-center justify-between text-[#FF5353]">
+  <div class="mt-4 pb-4 first:mt-0">
+    <div class="flex items-center justify-between">
       <h3 class="text-left text-base">
         Transaction {{ transactionIndex + 1 }}
       </h3>
-      <button @click="emit('removeTransaction', transactionIndex)">
-        Remove
+      <button
+        class="p-[6px] transition-colors duration-200 group"
+        @click="emit('removeTransaction', transactionIndex)"
+      >
+        <BaseIcon
+          class="text-red/80 group-hover:text-red"
+          name="close"
+          size="14"
+        />
```

Comment:
<img width="593" alt="Untitled 2" src="https://github.com/snapshot-labs/snapshot/assets/15967809/4fd88786-478b-40ee-b2c0-bd895ab355d4">

The icon is not too noticeable, could we also add the text "Remove" along with icon?